### ISSUE TITLE
GitHub footer elements does not Have Discernible Name

### ISSUE THREAD

**Issue by pdehaan** [mozilla/blurts-server#836]
![Firefox_Monitor___Scan_Results](https://user-images.githubusercontent.com/557895/54240530-8c119580-44db-11e9-986e-1068ee5b028b.png)

Not sure if we need to add some [`aria-label`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute) or something.

```html
<a data-ga-link="" data-event-category="Utility &amp; Outbound Links" data-event-label="GitHub" href="https://github.com/mozilla/blurts-server/" target="_blank" rel="noopener noreferrer">
    <svg id="GitHub" aria-hidden="true">
      <use xlink:href="/img/sprites.svg#GitHub"></use>
      <use href="/img/sprites.svg#GitHub"></use>
    </svg>
</a>
```
